status-im/status-legacy
# [developerXP] remove unused deps

- remove warning for asyncstorage deprecation
- remove unused asyncstorage-down library

### Steps to test

I couldn't find any usage for this library in the codebase so this shouldn't have change anything.
Only requires a basic sanity check
Tested on Android emulator, opening chat, wallet, settings and playing around.

status: ready
